CHANGELOG.md

```diff
@@ -57,6 +57,7 @@ All notable, unreleased changes to this project will be documented in this file.
 - Add an functionality to sort products by their "minimal variant price" - #4416 by @derenio
 - New stripe gateway implementation based on Stripe PaymentIntents API - #4606 by @salwator
 - Change AddressValidationRules API - #4655 by @Kwaidan00
+- Fix rendering price filter in Storefront 1.0 - #4677 by @maarcingebala
```

Comment:
I don't think this is needed as the `minimal_prices` changes were never released?